babel.config.js

```diff
@@ -51,23 +49,13 @@ module.exports = function(api) {
       break;
   }
 
-  if (includeRuntime) {
-    const babelRuntimePackageJSONPath = require.resolve(
-      "@babel/runtime/package.json"
-    );
-    const path = require("path");
-    transformRuntimeOpts = {
-      version: require(babelRuntimePackageJSONPath).version,
-      absoluteRuntime: path.dirname(babelRuntimePackageJSONPath),
-    };
-  }
-
   const config = {
     // Our dependencies are all standard CommonJS, along with all sorts of
     // other random files in Babel's codebase, so we use script as the default,
     // and then mark actual modules as modules farther down.
     sourceType: "script",
     comments: false,
+    exclude: "**/node_modules/**",
```

Comment:
What is the reason for this change? The comment above implies that this config is mainly meant for `node_modules`.

Reply:
Oh good catch! It was meant for to deal with `'import' and 'export' may appear only with 'sourceType: "module"'` since rollup commonjs converts them into ESM, but apparently it is an overkill.